apps/demo-app/src/app/pages/dialog-sample/dialog-sample.component.html

```diff
@@ -60,6 +60,23 @@
     </sk-dialog>
   </div>
 
+  <div>
+    <button (click)="isDialogWithFallbackButtonOpen = true">
+      open Dialog with fallback outer button right
+    </button>
+
+    <sk-dialog
+      [(open)]="isDialogWithFallbackButtonOpen"
+      (close)="isDialogWithFallbackButtonOpen = false"
+      [showCloseButton]="true"
+    >
+      <div>
+        here comes some content. This is a dialog with the a fallback Close
+        Button. (when the user don't have a close button)
+      </div>
+    </sk-dialog>
+  </div>
+
```

Comment:
would be nice if you add an example with inner fallback close button as well, i think the inner should be default as well

Reply:
k, added example with inner fallback close button. Dont know if we should set inner as default. Our approach was to set every input to false and outer was the default. I think we could discuss it quickly in a meet and change it if we want to but maybe in another PR cause its not related to the fallback.